transitions/index.js

```diff
@@ -29,70 +29,35 @@ var availableTransitions = [
   'update_sent_forms'
 ];
 
-/*
- * Supported configuration:
```

Comment:
Ok, I basically converted that documentation into tests. But I think you're right both are useful.  I should probably add something to the actual docs/readme...